### DISCUSSION TITLE
Is it possible to customise the default theme?

### DISCUSSION THREAD

**Discussion by StanSvec** [JanDeDobbeleer/oh-my-posh#2961] [Troubleshoot] [answered]
I like the default theme and need only to tweak it a bit, but I cannot find or generate the config file. `oh-my-posh config export` gives only link to the JSON schema and not to the actual config:
```
{
  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json"
}
```
Is the config file available anywhere? Thanks!

**Comment by JanDeDobbeleer** [comment, marked as answer]
@StanSvec that does seem like a bug due to now shipping the config internally. That said, you can still find it [here](https://github.com/JanDeDobbeleer/oh-my-posh/blob/ac0ffec43a5fd278c68573a09afd036b9acd1ad8/themes/default.omp.json).

**Reply by JanDeDobbeleer** [reply]
Fixed (only for JSON).

**Reply by StanSvec** [reply]
I don't know how I missed that default config in themes directory. Thanks for pointing me!
